Puppetfile

```diff
@@ -28,7 +28,8 @@ mod 'puppetlabs/catalog_preview', :latest
 mod 'puppet/archive', :latest
 mod 'puppetlabs/inifile', :latest
 mod 'graduway/ec2tagfacts',
-  :git => 'https://github.com/graduway/ec2tagfacts',
+  :git => 'https://github.com/graduway/ec2tagfacts', 
+  :branch => 'master'
```

Comment:
`master` is the default according to https://puppet.com/docs/pe/2017.3/code_management/puppetfile.html#declare-content-from-a-relative-control-repo-branch, please remove.